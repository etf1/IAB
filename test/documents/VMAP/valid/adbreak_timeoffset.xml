<!-- Different values for timeOffset on AdBreak -->
<vmap:VMAP version='1.0' xmlns:vmap='http://www.iab.net/vmap-1.0'>
    <vmap:AdBreak breakId='cue_point-347762684' breakType='linear' timeOffset="00:59:12">
        <vmap:AdSource allowMultipleAds='true' followRedirects='true' id='1' >
            <vmap:VASTAdData>
                <VAST version='3.0'
                      xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
                      xsi:noNamespaceSchemaLocation='vast.xsd'>
                </VAST>
            </vmap:VASTAdData>
        </vmap:AdSource>
    </vmap:AdBreak>
    <vmap:AdBreak breakId='cue_point-347762684' breakType='linear' timeOffset="00:59:12.123">
        <vmap:AdSource allowMultipleAds='true' followRedirects='true' id='1' >
            <vmap:VASTAdData>
                <VAST version='3.0'
                      xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
                      xsi:noNamespaceSchemaLocation='vast.xsd'>
                </VAST>
            </vmap:VASTAdData>
        </vmap:AdSource>
    </vmap:AdBreak>
    <vmap:AdBreak breakId='cue_point-347762684' breakType='linear' timeOffset="50%">
        <vmap:AdSource allowMultipleAds='true' followRedirects='true' id='1' >
            <vmap:VASTAdData>
                <VAST version='3.0'
                      xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
                      xsi:noNamespaceSchemaLocation='vast.xsd'>
                </VAST>
            </vmap:VASTAdData>
        </vmap:AdSource>
    </vmap:AdBreak>
    <vmap:AdBreak breakId='cue_point-347762684' breakType='linear' timeOffset="start">
        <vmap:AdSource allowMultipleAds='true' followRedirects='true' id='1' >
            <vmap:VASTAdData>
                <VAST version='3.0'
                      xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
                      xsi:noNamespaceSchemaLocation='vast.xsd'>
                </VAST>
            </vmap:VASTAdData>
        </vmap:AdSource>
    </vmap:AdBreak>
    <vmap:AdBreak breakId='cue_point-347762684' breakType='linear' timeOffset="end">
        <vmap:AdSource allowMultipleAds='true' followRedirects='true' id='1' >
            <vmap:VASTAdData>
                <VAST version='3.0'
                      xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
                      xsi:noNamespaceSchemaLocation='vast.xsd'>
                </VAST>
            </vmap:VASTAdData>
        </vmap:AdSource>
    </vmap:AdBreak>
    <vmap:AdBreak breakId='cue_point-347762684' breakType='linear' timeOffset="#8">
        <vmap:AdSource allowMultipleAds='true' followRedirects='true' id='1' >
            <vmap:VASTAdData>
                <VAST version='3.0'
                      xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
                      xsi:noNamespaceSchemaLocation='vast.xsd'>
                </VAST>
            </vmap:VASTAdData>
        </vmap:AdSource>
    </vmap:AdBreak>
</vmap:VMAP>
