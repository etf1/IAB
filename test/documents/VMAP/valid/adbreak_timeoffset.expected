{
  "version": "1.0",
  "breaks": [
    {
      "timeOffset": "00:59:12",
      "breakTypes": [1],
      "source": {
        "id": "1",
        "allowMultipleAds": true,
        "followRedirects": true,
        "dataType": 1,
        "VASTAdData": {
          "version": "3.0",
          "ads": []
        }
      },
      "id": "cue_point-347762684"
    },
    {
      "timeOffset": "00:59:12.123",
      "breakTypes": [1],
      "source": {
        "id": "1",
        "allowMultipleAds": true,
        "followRedirects": true,
        "dataType": 1,
        "VASTAdData": {
          "version": "3.0",
          "ads": []
        }
      },
      "id": "cue_point-347762684"
    },
    {
      "timeOffset": "50%",
      "breakTypes": [1],
      "source": {
        "id": "1",
        "allowMultipleAds": true,
        "followRedirects": true,
        "dataType": 1,
        "VASTAdData": {
          "version": "3.0",
          "ads": []
        }
      },
      "id": "cue_point-347762684"
    },
    {
      "timeOffset": "start",
      "breakTypes": [1],
      "source": {
        "id": "1",
        "allowMultipleAds": true,
        "followRedirects": true,
        "dataType": 1,
        "VASTAdData": {
          "version": "3.0",
          "ads": []
        }
      },
      "id": "cue_point-347762684"
    },
    {
      "timeOffset": "end",
      "breakTypes": [1],
      "source": {
        "id": "1",
        "allowMultipleAds": true,
        "followRedirects": true,
        "dataType": 1,
        "VASTAdData": {
          "version": "3.0",
          "ads": []
        }
      },
      "id": "cue_point-347762684"
    },
    {
      "timeOffset": "#8",
      "breakTypes": [1],
      "source": {
        "id": "1",
        "allowMultipleAds": true,
        "followRedirects": true,
        "dataType": 1,
        "VASTAdData": {
          "version": "3.0",
          "ads": []
        }
      },
      "id": "cue_point-347762684"
    }
  ]
}
